src/cryptography/hazmat/backends/openssl/rsa.py

```diff
@@ -559,6 +565,62 @@ def private_numbers(self):
             )
         )
 
+    def as_bytes(self, encoding, fmt, encryption_algorithm):
```

Comment:
Yep. If we're comfortable with it shadowing the builtin for this method I am okay with using format though.